include/envoy/upstream/load_balancer_type.h

```diff
@@ -25,6 +25,14 @@ enum class LoadBalancerType {
   ClusterProvided
 };
 
+struct SubsetSelector {
+  std::set<std::string> selector_keys_;
+  envoy::api::v2::Cluster::LbSubsetConfig::LbSubsetSelector::LbSubsetSelectorFallbackPolicy
+      fallback_policy_;
+};
+
+using SubsetSelectorPtr = std::shared_ptr<SubsetSelector>;
```

Comment:
nit: I think our style is to use a typedef here.

Reply:
Oops. I didn't @htuch's earlier comment re: using. I shouldn't have asked you to change it. Feel free to leave it as typedef.

Reply:
@zuercher switched back to `using` :)

Reply:
Thank you!